shank-idl/src/file.rs

```diff
@@ -40,6 +41,7 @@ impl Default for ParseIdlConfig {
             program_name: Default::default(),
             detect_custom_struct: Default::default(),
             require_program_address: true,
+            override_program_address: None,
```

Comment:
```suggestion
            program_address_override: None,
```
and all related renames.